authaad.properties

```diff
@@ -2,14 +2,20 @@ name=Azure Active Directory Authentication for SonarQube
 category=Integration
 description=Allows the use of Azure Active Directory as an authentication source for SonarQube.
 homepageUrl=https://github.com/hkamel/sonar-auth-aad
-archivedVersions=
-publicVersions=1.2.0
+archivedVersions=1.2.0
+publicVersions=1.3.1
 
 defaults.mavenGroupId=org.almrangers.auth.aad
 defaults.mavenArtifactId=sonar-auth-aad-plugin
 
+1.3.1.description=Client credential flow, updated settings layout, support for localization.
+1.3.1.sqVersions=[8.9, LATEST]
+1.3.1.date=2022-08-02
+1.3.1.changelogUrl=https://github.com/hkamel/sonar-auth-aad/releases/tag/1.3.1
+1.3.1.downloadUrl=https://github.com/hkamel/sonar-auth-aad/releases/download/1.3.1/sonar-auth-aad-plugin-1.3.1.jar
+
 1.2.0.description=Group sync enhancements including transitive members. Code cleanup.
-1.2.0.sqVersions=[8.9, LATEST]
+1.2.0.sqVersions=[8.9]
```

Comment:
Interesting. The community post on the process doesn't note this, but simply says that if the version is "archived" and installed, then users will get a prompt to upgrade to the latest compatible current version. It might be good to revise that language to clarify.